packages/terra-notification-dialog/src/NotificationDialog.jsx

```diff
@@ -66,31 +92,62 @@ const propTypes = {
    * Toggle to show notification-dialog or not.
    */
   isOpen: PropTypes.bool.isRequired,
+  /**
+   * Reverses the order of notification action buttons
+   */
+  buttonOrder: PropTypes.oneOf([
+    'acceptFirst',
+    'rejectFirst',
+  ]).isRequired,
+  /**
+   * Determines whether acceptAction, rejectAction or neither is emphasizedAction
+   */
+  emphasizedAction: PropTypes.oneOf([
+    'none',
+    'accept',
+    'reject',
+  ]),
 };
 
 const defaultProps = {
   title: null,
-  message: null,
+  startMessage: null,
+  endMessage: null,
+  content: null,
   variant: variants.CUSTOM,
+  emphasizedAction: 'none',
 };
 
-const actionSection = (primaryAction, secondaryAction) => {
-  let actionButton = null;
-  let dismissButton = null;
-  if (!primaryAction && !secondaryAction) {
+const actionSection = (acceptAction, rejectAction, buttonOrder, emphasizedAction) => {
+  let acceptButton = null;
+  let rejectButton = null;
+  if (!acceptAction && !rejectAction) {
     return null;
   }
-  if (primaryAction) {
-    actionButton = <Button text={primaryAction.text} variant={Button.Opts.Variants.EMPHASIS} onClick={primaryAction.onClick} />;
+  if (acceptAction) {
+    acceptButton = (emphasizedAction === 'accept')
+      ? <Button text={acceptAction.text} variant={Button.Opts.Variants.EMPHASIS} onClick={acceptAction.onClick} />
```

Comment:
can we use the string or variants object instead of opts?

Reply:
Switched to importing ButtonVariants here https://github.com/cerner/terra-framework/pull/899/commits/cd5d87726ffa459e43be295823a935fea67d1f7a?file-filters%5B%5D=.jsx&file-filters%5B%5D=.md#diff-cb62dc7ce0197bc569a9952450bb9fe5R130